PulsePlayer/PulsePlayer.xcodeproj/project.pbxproj

```diff
@@ -553,7 +563,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "# bash \"${BUILT_PRODUCTS_DIR}/${FRAMEWORKS_FOLDER_PATH}/Pulse.framework/strip-frameworks.sh\"";
+			shellScript = "# bash \"${BUILT_PRODUCTS_DIR}/${FRAMEWORKS_FOLDER_PATH}/Pulse.xcframework/strip-frameworks.sh\"\n";
```

Comment:
@nichenriksson  Although I think this step is not needed. For now, I have just updated to at least have the correct location.